lib/thrift.ex

```diff
@@ -7,4 +7,10 @@ defmodule Thrift do
 
     * `Thrift.Parser` - functions for parsing `.thift` schema files
   """
+
+  @type i8   :: (-128..127)
+  @type double :: float()
```

Comment:
Probably pedantic, but move `double` to the end of the list so it's not in the middle of the integer types?